learning-progression-with-la-schema.md

```diff
@@ -1058,6 +1058,10 @@ Program state. List with examples.
 ##### Clock cycles
 [[toc](#table-of-contents)]
 
+Synchronous circuits are circuits in which a clock governs when a circuit will change to its next state. This coordination is necessary in complex circuits like CPUs, because some operations may take longer to complete than others. This can create a bug known as a [race condition](https://en.wikipedia.org/wiki/Race_condition), in which erroneous output is produced because the timing of the circuit's change to the next state was not consistent throughout the circuit.
```

Comment:
@iconoptic, you need to pull back (i.e. start a bit farther back):
1. Talk briefly about state and memory.
2. You should mention the simplest stateful circuit, which is the _flip-flop_. 
3. The ff has two states and that's where we connect hardware with binary. Show a simplified D-ff truth table. Refer back to the [1-bit of memory explanation](https://github.com/ivogeorg/ce-learning-progression-004-computation-and-io/blob/master/learning-progression-with-la-schema.md#addressing). 
4. Add the clock line to the D-ff. Use this as a segue to synchronous circuits.

Reply:
@iconoptic, the clock synchronizes all stateful circuit components to _change their state at the same time_.